Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.h

```diff
@@ -77,13 +78,18 @@ namespace WhiteBox
         //! Enter the sub-mode for edge restore.
         void EnterEdgeRestoreMode();
 
+        //! Enter the sub-mode for transforming
+        void EnterTransformMode();
+
```

Comment:
uber nit: Might be nice to group all the `Enter` functions together so I'd lose the line break on line 80